src/lib.rs

```diff
@@ -37,7 +37,9 @@ pub use generators::{Generators, GeneratorsView, PedersenGenerators};
 pub use proof_transcript::ProofTranscript;
 pub use range_proof::RangeProof;
 
-/// API for performing the aggregate-proving multiparty computation protocol.
+/// The `aggregation` module contains API for performing the aggregated multiparty computation protocol.
+///
+#[doc(include = "../docs/aggregation-api.md")]
```

Comment:
I'm not sure what the order is for combining doc comments and doc includes, and I can't see the sentence in the rendered docs, so it's probably better to have no doc comment and just put the text right into the API md file